cloudinit/distros/__init__.py

```diff
@@ -998,6 +1001,26 @@ def do_as(self, command: list, user: str, cwd: str = "", **kwargs):
             **kwargs,
         )
 
+    @staticmethod
+    def build_dhclient_cmd(
+        path: str,
+        lease_file: str,
+        pid_file: str,
+        interface: str,
+        config_file: str,
+    ) -> list:
+        return [
+            path,
+            "-1",
+            "-v",
+            "-lf",
+            lease_file,
+            "-pf",
+            pid_file,
+            "-sf",
+            "/bin/true",
+        ] + (["-cf", config_file, interface] if config_file else [interface])
```

Comment:
No need for parens in this line
```suggestion
        ] + ["-cf", config_file, interface] if config_file else [interface]
```

Reply:
I disagree. That would cause an invalid command to be returned when `config_file` is passed. Note the `if`/`else` logic.

Reply:
Man, note to self, do not review on a plane w/ lack of sleep. Thanks precedence. disregard please.